client/src/pages/MoviePage.js

```diff
@@ -1,12 +1,173 @@
+		rating: 7
+	},
+	{
+		user: 'ur mom',
+		content: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Integer cursus ut elit vel ullamcorper. Nam sagittis tincidunt egestas. Curabitur pretium erat et eros suscipit pretium. Sed blandit malesuada nisl in finibus. Vivamus mi quam, mollis vel lorem quis, placerat maximus turpis. Fusce tempus mi ligula, eu fringilla augue laoreet a. Vivamus elementum ex vel ante sollicitudin volutpat. Morbi ultrices tincidunt fringilla.',
+		rating: 7
+	},
+	{
+		avatar: 'https://lh3.googleusercontent.com/a-/AOh14GjJNN27t2gdzWaXWwJeG_feC0VeL0u6ahnOPTsi=s96-c',
+		user: 'ur mom',
+		content: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Integer cursus ut elit vel ullamcorper. Nam sagittis tincidunt egestas. Curabitur pretium erat et eros suscipit pretium. Sed blandit malesuada nisl in finibus. Vivamus mi quam, mollis vel lorem quis, placerat maximus turpis. Fusce tempus mi ligula, eu fringilla augue laoreet a. Vivamus elementum ex vel ante sollicitudin volutpat. Morbi ultrices tincidunt fringilla.',
+		rating: 7
+	},
+	]
+
+	//Movie review ... may change later
+	let movieReview
+	if (reviews && reviews.length > 2) {
+		movieReview = reviews.slice(0, 2);
+	} else {
+		movieReview = reviews
+	}
+
+	const showAllReviews = () => {
+		movieReview = reviews;
+	}
 
 	return (
 		<div>
+			<div className="fixed max-w-full inset-x-0 z-20">
+				<Navbar />
+			</div>
+			{!movieDetails ?
+				<div>
+					<Loading />
+				</div> :
+
+				<div className="py-24 flex justify-center">
+					<div className='items-center max-w-3/4 flex justify-center flex-col'>
+						<div className="max-w-1/2 items-center text-center">
+							<div className="flex mt-1 justify-center p-5 max-w-1/2">
+								<div className="w-full lg:w-1/5 lg:rounded-r max-w-screen max-h-screen object-cover">
+									<img src={`https://image.tmdb.org/t/p/w500${movieDetails.poster_path}`} className="" alt={movieDetails.title} />
+								</div>
+
+								<div className="pl-3 w-full lg:w-4/5">
+									<h3 className="font-bold text-lg text-left md:text-md">{movieDetails.title}</h3>
+									<h5 className="text-xs text-left mb-4">{movieDetails.release_date}</h5>
+									<div className="">
+										<p className='text-sm flex items-center'>Rating: {movieDetails.vote_average}<StarIcon className="h-4 w-4 text-yellow-500" /> </p>
+									</div>
+									<p className='h-auto text-left text-sm'>{movieDetails.overview}</p>
+
+									<div className="flex text-xs mt-3"><p className="text-xs underline">Genres:</p>
+										{movieDetails.genres.map(genre => (
+											<p className='text-xs ml-4 no-underline' key={genre.id}>{genre.name};</p>
+										))}
+									</div>
+
+									<Link className="text-xs text-left mt-2" to={movieDetails.homepage}>
+										<p className="mt-2 text-indigo-400 hover:text-blue-800 hover:underline">HomePage</p>
+									</Link>
```

Comment:
When you're referring to external pages, it should always be anchor tag. So something like this `<a class=''>...</a>`. `Link` won't work properly